pyogrio/tests/test_raw_io.py

```diff
@@ -299,3 +299,65 @@ def test_write_unsupported(tmpdir, naturalearth_lowres):
 
     with pytest.raises(DataSourceError, match="does not support write functionality"):
         write(filename, geometry, field_data, driver="OpenFileGDB", **meta)
+
+
+def assert_equal_result(result1, result2):
+    meta1, index1, geometry1, field_data1 = result1
+    meta2, index2, geometry2, field_data2 = result2
+
+    assert np.array_equal(meta1["fields"], meta2["fields"])
+    assert np.array_equal(index1, index2)
+    # assert np.array_equal(geometry1, geometry2)
```

Comment:
minor nit: remove commented line

Reply:
Turned it into a small explanation why we are using pygeos here